### PR TITLE
Purchase a download through PayPal with no title results in empty car…

### PR DESCRIPTION
**Pull request by PareshRadadiya**
set the PayPal item_name field to the download ID if no title is present

### CODE REVIEW

File: includes/cart/functions.php [hunk 1]
```diff
@@ -558,6 +558,28 @@ function edd_get_cart_item_price_name( $item = array() ) {
 }
 
 /**
+ * Get cart item title
+ *
+ * @since 2.4.2
```
Comment (pippinsplugins) [inline includes/cart/functions.php]: This should be 4.2

**Comment by pippinsplugins** [comment]
This looks like a good answer to me. We'll test further and merge in. Thanks!

**Comment by cklosowski** [comment]
@PareshRadadiya Can you please create your changes in a new branch (from `master`) and submit a PR that will merge cleanly?

**Comment by PareshRadadiya** [comment]
@cklosowski  Yep sure

**Comment by PareshRadadiya** [comment]
#3596

**Pull request closed without merging**